addons/payment_authorize/models/payment_provider.py

```diff
@@ -133,3 +140,73 @@ def _authorize_get_inline_form_values(self):
             'client_key': self.authorize_client_key,
         }
         return json.dumps(inline_form_values)
+
+    def action_create_webhook(self):
+        """Create or update a webhook in Authorize.Net for transaction notifications.
+
+        Note: self.ensure_one()
+
+        :return: A notification action to display the result
+        :rtype: dict
+        """
+        self.ensure_one()
+
+        if self.state == 'disabled':
+            raise UserError(_("Cannot create webhook while the provider is disabled."))
+
+        authorize_api = AuthorizeAPI(self)
+        base_url = self.get_base_url()
+        if base_url.startswith("http://"):
+            base_url = "https://" + base_url[len("http://"):]
+
+        webhook_url = f"{base_url.rstrip('/')}/payment/authorize/webhook"
```

Comment:
Is there a reason for not doing this?
```suggestion
        webhook_url = tools.urls.urljoin(self.get_base_url(), '/payment/authorize/webhook')
```